contactaccessrights/js/src/access-rights/models/location.js

```diff
@@ -0,0 +1,14 @@
+define([
+  'access-rights/modules/models',
+  'access-rights/services/location',
+  'common/services/api',
+  'common/models/model'
+], function (models) {
+  'use strict';
+
+  models.factory('Location', ['Model', 'api.location', function (Model, api) {
+    return Model.extend({
+      getAll: api.query.bind(api)
```

Comment:
As we'd talked about keeping each API on its own extension, I preferred to do it like that.